apps/src/templates/feedback/Code.jsx

```diff
@@ -0,0 +1,17 @@
+var msg = require('../../locale');
+
+var Code = React.createClass({
+  
+  propTypes: {
+    message: React.PropTypes.string.isRequired,
+    code: React.PropTypes.string.isRequired,
+  },
+
+  render: function () {
+    return (<div className="generated-code-container">
+      <p className="generatedCodeMessage" dangerouslySetInnerHTML={{__html: this.props.message}} />
+      <pre className="generatedCode" dangerouslySetInnerHTML={{ __html: this.props.code }} />
+    </div>);
```

Comment:
Can we replace these classNames with inlined styles? It looks like generatedCodeMessage does have some hover styling, but maybe that's insignificant enough that we don't mind losing it?

FWIW, I've explored adding radium, which would allow us to do hover for inlined styles. The main reason I'm somewhat wary to add it is it does increase our file size some (an extra 50k or so I think)